src/main/java/com/icthh/xm/ms/balance/web/rest/BalanceResource.java

```diff
@@ -140,8 +142,22 @@ public BalanceResource(BalanceService balanceService, BalanceQueryService balanc
     @PreAuthorize("hasPermission({'reloadRequest': #reloadRequest}, 'BALANCE.RELOAD')")
     @PostMapping("/balances/reload")
     @Timed
-    public ResponseEntity<BalanceDTO> createBalance(@Valid @RequestBody ReloadBalanceRequest reloadRequest) {
+    public void createBalance(@Valid @RequestBody ReloadBalanceRequest reloadRequest) {
```

Comment:
Maybe not right method name for mapping /balances/reload

Reply:
yes, it's mistake

Reply:
I meant not reloadBalance instead of createBalance